src/systemctl/systemctl-show.c

```diff
@@ -821,18 +821,37 @@ static void print_status_info(
 }
 
 static void show_unit_help(UnitStatusInfo *i) {
+        bool previous_man_page = false;
+
         assert(i);
 
         if (!i->documentation) {
                 log_info("Documentation for %s not known.", i->id);
                 return;
         }
 
-        STRV_FOREACH(p, i->documentation)
-                if (startswith(*p, "man:"))
-                        show_man_page(*p + 4, false);
-                else
-                        log_info("Can't show: %s", *p);
+        STRV_FOREACH(doc, i->documentation) {
+                const char *p;
+                _cleanup_free_ char *t = NULL;
```

Comment:
Not a big matter, but the declaration of `t` can be moved to the `else` block below.